### PR TITLE
Vector of shared ptrs

### PR DESCRIPTION
**Pull request by kostuch76**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: vector-of-shared-ptrs
🏅 Granted 5 XP for @kostuch76

File: homework/vector-of-shared-ptrs/vectorFunctions.cpp [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#include <iostream>
+#include <vector>
+#include <memory>
+#include "vectorFunctions.hpp"
+
+std::vector<std::shared_ptr<int>> generate(int count)
+{
+	std::vector<std::shared_ptr<int>> v;
```
Comment (nauka-programowania-MA) [inline homework/vector-of-shared-ptrs/vectorFunctions.cpp:8]: reserve memory

File: homework/vector-of-shared-ptrs/vectorFunctions.cpp [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	for (int i = 0; i < count; ++i)
+	{
+		auto ptr = std::make_shared<int>(i);
+		v.push_back(ptr);
```
Comment (nauka-programowania-MA) [inline homework/vector-of-shared-ptrs/vectorFunctions.cpp:12]: create ptr here

File: homework/vector-of-shared-ptrs/vectorFunctions.cpp [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	}
+	return v;
+}
+void print(const std::vector<std::shared_ptr<int>>& v)
+{
+    for (const auto& el: v)
+    {
+        std::cout << *el << ' ';
```
Comment (nauka-programowania-MA) [inline homework/vector-of-shared-ptrs/vectorFunctions.cpp:20]: check if ptr is != nullptr

File: homework/vector-of-shared-ptrs/vectorFunctions.cpp [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+void add10(const std::vector<std::shared_ptr<int>>& v)
+{
+    for (const auto& el : v)
+    {
+	if (el != nullptr)
+	{
+		*el += 10;	
+	}
+	
+    }
```
Comment (nauka-programowania-MA) [inline homework/vector-of-shared-ptrs/vectorFunctions.cpp:33]: Indentations.

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**